closed/autoconf/custom-hook.m4

```diff
@@ -317,20 +317,7 @@ AC_DEFUN_ONCE([OPENJDK_VERSION_DETAILS],
   # Second, (stable) sort on (W), (X):
   tag_sort2="$SORT -t. -k2n -k3n -s"
 
-  OPENJDK_TAG=`git -C $TOPDIR tag --points-at HEAD | $GREP "jdk-13.*+" | $GREP -v _openj9 | $tag_sort1 | $tag_sort2 | $TAIL -1`
-
-  # If there's no tag for HEAD, find the SHA of most recent ancestor that is tagged.
-  if test x$OPENJDK_TAG = x ; then
-    # For precision, get the full SHA for HEAD.
-    head_sha=`git -C $TOPDIR rev-parse HEAD`
-    # We insert 'filler' here so $head_sha will never be on the first line, which would break the sed filter.
-    tagged_sha=`($ECHO filler ; git -C $TOPDIR rev-list '--tags=jdk-13*+*' '--exclude=*_openj9*' --topo-order --no-walk HEAD) | $SED -e "1,/$head_sha/d" | $HEAD -1`
-
-    if test x$tagged_sha != x ; then
-      # Select the latest tag, like above.
-      OPENJDK_TAG=`git -C $TOPDIR tag --points-at $tagged_sha | $GREP "jdk-13.*+" | $GREP -v _openj9 | $tag_sort1 | $tag_sort2 | $TAIL -1`
-    fi
-  fi
+  OPENJDK_TAG=`git -C $TOPDIR tag --merged HEAD "jdk-13*+*" | $GREP -v _openj9 | $tag_sort1 | $tag_sort2 | $TAIL -1`
```

Comment:
`--merged` requires a newer version of git than I have on my Ubuntu 14.04 system.

Even on a system that understands that option I get:
> fatal: --merged and --no-merged option are only allowed with -l

Reply:
Adding `--list` to the command
> git -C $TOPDIR tag --merged HEAD "jdk-13*+*"

to get
> git -C $TOPDIR tag --merged HEAD --list "jdk-13*+*"

fixes the fatal error.

Reply:
> --merged requires a newer version of git than I have on my Ubuntu 14.04 system.

Not a blocker since we don't support Ubuntu 14.

The z/OS problem is a blocker. https://github.ibm.com/runtimes/infrastructure/issues/1783